doc/api/http.md

```diff
@@ -1788,7 +1788,7 @@ changes:
     `ServerResponse`.
 - `requestListener` {Function}
 
-* Returns: {http.Server}
+- Returns: {http.Server}
```

Comment:
I think we should be using asterisks instead for consistency. This might mean the `options` and `requestListener` lines should be changed instead.